### DISCUSSION TITLE
Slack access for metricstream teams

### DISCUSSION THREAD

**Discussion by kiranhariharan78** [ocsf/ocsf-docs#50] [Slack Workspace Requests]
Hi - we are looking to join the group to leverage the schema more in our product for cyber risk and compliance management 

email addresses are 
Kiran.hariharan.it@gmail.com
Kiran.hariharan@meteicstream.com
Rajkumar.oppilamani@metricstream.com
Nivedita.rudrappa@metricstream.com

**Comment by pagbabian-splunk** [comment]
Thank you! Your invitations have been sent, please introduce yourselves in the #general channel when you join.
